app/middleware/template-locals.js

```diff
@@ -1,10 +1,27 @@
+/*
+Examples:
+
+ - yes_no(is_admin) // => 'Yes' / 'No'
+ - yes_no(access_level, 'readwrite') // => 'Yes' / 'No'
+ - yes_no(is_admin, true, '(Admin)', '') // => '(Admin)' / ''
+ - yes_no(access, 'readwrite', 'Revoke', 'Grant') // => 'Revoke' / 'Grant'
+*/
+function yes_no(value, true_value=true, yes_str='Yes', no_str='No') {
+  if (value === true_value) {
+    return yes_str;
+  } else {
+    return no_str;
+  }
+}
```

Comment:
This should be a [template macro](https://mozilla.github.io/nunjucks/templating.html#macro), not a local function.